### PR TITLE
Feature/bazooka/GEN-1004: Claims Card

### PR DESCRIPTION
**Pull request by julia-andersson**
## [GEN-1004]

- Migrate claims from 🦒 to 🐙
- Updated design according to [figma](https://www.figma.com/file/qUhLjrKl98PAzHov9ilaDH/Hedvig-UI-Kit?node-id=21%3A440&mode=dev)

**Note:** I skipped payout amount for now since we do not have a value for this now (and it did not work in i Giraffe either so we still have the same functionality as before). This will be added later as soon we have all of this.

## Checklist

- [x] Dark/light mode verification
- [x] iPad verification
- [x] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Models/ClaimModel.swift [hunk 1]
```diff
@@ -1,203 +1,131 @@
...
-        pills: [ClaimModel.ClaimPill],
-        segments: [ClaimModel.ClaimStatusProgressSegment],
-        title: String,
-        subtitle: String,
-        claimDetailData: ClaimDetailData
+        status: ClaimStatus,
+        outcome: ClaimOutcome,
+        submittedAt: String?,
+        closedAt: String?,
+        signedAudioURL: String,
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Models/ClaimModel.swift:13]: make signedAudioURL optional

File: Projects/Claims/Sources/Models/ClaimModel.swift [hunk 2]
```diff
@@ -1,203 +1,131 @@
...
+        statusParagraph: String,
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Models/ClaimModel.swift:14]: dont need this in init -> statusParagraph

File: Projects/Claims/Sources/Models/ClaimModel.swift [hunk 3]
```diff
@@ -1,203 +1,131 @@
...
-            }
-            self.statusParagraph = claim.statusParagraph
-            self.type = claim.type ?? ""
-            self.payout = .init(amount: claim.payout?.amount ?? "", currency: claim.payout?.currency ?? "")
+    public let id: String
+    public let status: ClaimStatus
+    public let outcome: ClaimOutcome
+    public let submittedAt: String?
+    public let closedAt: String?
+    public let signedAudioURL: String
```
Comment (sladan-hedvig) [inline Projects/Claims/Sources/Models/ClaimModel.swift:47]: This is actually optional audio

**Review by sladan-hedvig — CHANGES_REQUESTED** [review]

File: Projects/Claims/Sources/Models/ClaimModel.swift (hunk 2 above)
Comment (julia-andersson) [reply Projects/Claims/Sources/Models/ClaimModel.swift:14]: will remove this

**Review by sladan-hedvig — APPROVED** [review]
Looks great!

**Pull request merged**